packages/builder/src/pages/builder/portal/overview/_components/SettingsTab.svelte

```diff
@@ -27,18 +28,24 @@
 <div class="settings-tab">
   <Page wide={false}>
     <Layout gap="XL" paddingY="XXL" paddingX="">
+      <Layout gap="XS" noPadding>
+        <Heading size="M">App settings</Heading>
```

Comment:
NAB: A subjective point, but imo this heading is redundant because you can see that the settings tab is highlighted and selected.
I like the 'App details' header though.